mod/settings.php

```diff
@@ -1277,7 +1277,7 @@ function settings_content(App $a) {
 
 		'$h_pass' 	=> t('Password Settings'),
 		'$password1'=> array('password', t('New Password:'), '', ''),
-		'$password2'=> array('confirm', t('Confirm:'), '', t('Leave password fields blank unless changing')),
```

Comment:
I just found the answer to my question and can now see why this was flag up the past.

Yes, in Vier and Co. themes the sections are somehow link (even they have already separate saving buttons). Whereas in Frio this is not the case and each section is "free-standing", making a slightly absurd hint.